app/models/chargeback_rate_detail.rb

```diff
@@ -225,6 +229,10 @@ def contiguous_tiers?
 
   private
 
+  def gratis?
+    chargeback_tiers.all?(&:gratis?)
```

Comment:
Right.

Although, that's only true for `ChargebackTiers` that have been saved after the validation was introduced. The validation check for this is not here from day 1 I think.

Anyway, I prefer this way for readability reasons.